packages/nodes-base/nodes/Odoo/Odoo.node.ts

```diff
@@ -116,19 +116,19 @@ export class Odoo implements INodeType {
 				const userID = await odooGetUserID.call(this, db, username, password, url);
 
 				const response = await odooGetModelFields.call(this, db, userID, password, resource, url);
-				const options = Object.values(response).map((field) => {
+				const options = Object.entries(response).map(([key, field]) => {
 					const optionField = field as { [key: string]: string };
 					let name = '';
 					try {
 						name = capitalCase(optionField.name);
 					} catch (error) {
-						name = optionField.name;
+						optionField.name = optionField.string;
```

Comment:
Could you either:
- assign this to `name` and use that variable below?
- on L123 assign to `optionField.name` and remove the name variable?

currently the name variables is not being used